browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3449,6 +3449,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_BRAVE_WALLET_SEARCHING_FOR_DOMAIN" desc="Searching for wallet domain progress message.">
       Searching for domain...
     </message>
+    <message name="IDS_REWARDS_TIPPING_SELECT_AMOUNT" desc="Select tipping amount text">
+      Please select tipping amount.
```

Comment:
@rmcfadden3 does this string look okay ? we show a toast message, when user don't have any amount selected for tipping.

Reply:
@deeppandya — it's okay. But for grammar, I wonder if it should say "Please select tip amount" or else "Please select amount to tip" instead.

Reply:
Thank you @rmcfadden3 i have updated the change with `Please select amount to tip`